index.html

```diff
@@ -27,13 +34,19 @@ <h1>POKEYOU</h1>
 <br>
     <!-- Screen 2 HTML -->
 
-    <label for="avatar">Choose a profile picture:</label>
+    <div id="User Image Container">
+    <label for="userImage">Choose a profile picture:</label>
 
-    <input type="file" id="avatar" name="avatar" accept="image/png, image/jpeg">
+    <input type="file" id="user-image" name="image" accept="image/png, image/jpeg">
+    </div>
 
-    <!-- Adding a submit button -->
+    <br>
+    <br>
 
+    <!-- Adding a submit button -->
+    <div id="Button Container">
```

Comment:
Is this 1 ID or 2?

Reply:
hyphens should be added to all containers now